### DISCUSSION TITLE
How get IP address??

### DISCUSSION THREAD

**Discussion by agracia-foticos** [nuxt/nuxt#26225] [Questions]
whats the best solution to get the request IP address in Nuxt 3??

In client side and server side.

Thanks in advance

**Comment by AloisSeckar** [comment]
For getting it on server, the best I have found yet is this npm package - https://www.npmjs.com/package/request-ip - including the different ways the use to determine it in Readme. But it is not getting new updates, so maybe there are alternatives I am not aware of.

Getting it on client, it doesnt quite make sense to me and it is generally unsafe, but here is some info about it - https://stackoverflow.com/questions/20194722/can-you-get-a-users-local-lan-ip-address-via-javascript
